Assignment_2/hexadecimalConversion/HexCalc.java

```diff
@@ -0,0 +1,355 @@
+package hexadecimalConversion;
+
+import java.util.Scanner;
+
+/**
+ * HexCalc is a public class which is heavily used in the main class HexCalc_Driver .
+ * 
+ * @author Aman Gautam
+ * 
+ * dated 15/07/2019
+ * 
+ */
+public class HexCalc {
+	private String firstnumber;
+	private String secondnumber;
+	
+	
+	Scanner sc = new Scanner(System.in);
+
+	
+	/**
+	 * this is a getDecimal method for the conversion of HexaDecimal no. to
+	 * Decimal no.
+	 * 
+	 * @param  {String}hexnumber   Hexadecimal no. user input
+	 * 
+	 * @return {integer}value     Decimal equivalent of Hexadecimal no
+	 */
+	public int getDecimal(String hexnumber) {
+		int index, value = 0;
+		char character;
+		String digits = "0123456789ABCDEF", string;

```

Comment:
Never use reserved keywords as a variable name

Reply:
changed to temp (string type)